paper-input-behavior.html

```diff
@@ -224,6 +224,13 @@
       },
 
       /**
+       * Bind this to the `<input is="iron-input">`'s `oninvalid` property.
+       */
+      oninvalid: {
+        type: String
+      },
+
+      /**
        * Limits the numeric or date-time increments.
        * Bind this to the `<input is="iron-input">`'s `step` property.
        */
```

Comment:
Thanks @notwaldorf and sorry for more n00b mistakes! Is PolymerElements/iron-input#70 what you meant?